src/Catalog/Helpers/PackageContentUriBuilder.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.Metadata.Catalog.Helpers
+{
+    /// <summary>
+    /// Utility class to validate package content URL format strings.
+    /// </summary>
+    public class PackageContentUriBuilder
+    {
+        public const string IdLowerPlaceholderString = "{id-lower}";
+        public const string VersionLowerPlaceholderString = "{version-lower}";
+
+        private const string NuPkgExtension = ".nupkg";
+        private readonly string _packageContentUrlFormat;
+
+        public PackageContentUriBuilder(string packageContentUrlFormat)
+        {
+            _packageContentUrlFormat = packageContentUrlFormat ?? throw new ArgumentNullException(nameof(packageContentUrlFormat));
+
+            if (!_packageContentUrlFormat.Contains("{id-lower}") || !_packageContentUrlFormat.Contains(VersionLowerPlaceholderString))
```

Comment:
>{id-lower}" [](start = 52, length = 11)

use the const you already defined

Reply:
Oops. Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/63879c738456c91777a2798a7f2e2e6b0e260224